### DISCUSSION TITLE
LFS consumption sky rocketed?

### DISCUSSION THREAD

**Discussion by shultays** [community/community#101757] [Repositories]
### Select Topic Area

Question

### Feature Area

Projects

### Body

2 days ago I got an email about my LFS bandwidth approaching its limit and these were the data

  Bandwidth: 0.83 GB / 1 GB (83%)
  Storage: 1.27 GB / 1 GB (127%)

I didn't think of it that much. but today I got another one.

  Bandwidth: 1.1 GB / 1 GB (110%)
  Storage: 1.27 GB / 1 GB (127%)

I don't use github that much other than storing some old code. I am not part of any projects that people actively pull etc. What is going on? Where is that usage is coming? I am definitely not using any bandwidth and I doubt I have 1.27gb of data (unsure)

Can't I see which individual repo/project using those?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @shultays ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by shultays** [reply]
Thank you very much
